influxql/ast.go

```diff
@@ -642,6 +642,36 @@ type SelectStatement struct {
 	FillValue interface{}
 }
 
+// HasDerivative returns true if one of the field in the statement is a
+// derivative aggregate
+func (s *SelectStatement) HasDerivative() bool {
```

Comment:
I don't follow why this function is needed. Why is it important if a field's name has "derivative" in there?